incubator/jenkins/templates/ingress.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     component: "{{.Release.Name}}-{{.Values.Master.Component}}"
     dns: route53
   annotations:
+{{- if .Values.Ingress.Annotations }}
```

Comment:
You miss looked.
There is updates of 2 charts

```
.Values.Ingress.Annotations
```  
for jenkins chart

```ingress:
   annotations: {}
   labels: {}
```
for lamp chart

Difference in notations because jenkins chart is a fork and I'd like to follow notation they use